README.MD

```diff
@@ -28,23 +32,6 @@ Run tests
 $ yarn test
 ```
 
-# Rebuild the semantic UI asserts
```

Comment:
We are still using semantic UI, so I think we should keep this instructions here. Maybe add a disclaimer that this is only needed if we need to upgrade semantic